### DISCUSSION TITLE
Promoting content in one section over another

### DISCUSSION THREAD

**Discussion by markzegarelli** [algolia/docsearch#2043] [Q&A] [answered]
Hello, I have two sections a site I've recently inherited:
- "Maintenance SDKs"
- "SDK Reference"

When I search for `Browser SDK`, the content from the Maintenance SDKs section appear first in the results. This is legacy content though, and I'd like for the newer content to appear at the top.

Is there any way I can boost the content in SDK Reference to always appear above Maintenance SDKs? I thought `pageRank` might be useful here, but I'm not sure if I understand it correctly, since it seems like it would apply equally to every result from the `pathsToMatch` property.

**Comment by shortcuts** [comment, marked as answer]
Hey @markzegarelli, indeed `pageRank` and `pathsToMatch` will be what you need to use here.

You can create a new `action` that only targets the legacy URLs, (so `pathsToMatch` would contain all legacy URLs), and add a negative `pageRank` to this action.

In the other action (the current one), just append the legacy URLs to the current `pathsToMatch`, prepended with a `!` (this means it's a negative pattern, so URLs are excluded)

don't hesitate to copy your config (without `appID` and `apiKey` fields) here if you need more help

**Reply by markzegarelli** [reply]
Thanks @shortcuts! I must have missed the option for multiple actions. I've updated my config and it seems to bee working as expected now.

**Comment by randombeeper** [comment]
I'm going to close this discussion as we are trying to keep interactions to our Discord at https://alg.li/discord and we'll be happy to help you there! This discussion area is not well monitored.
